webapi/webapi-nativefilesystem-xwalk-tests/nativefilesystem/Metadata_modificationTime.html

```diff
@@ -1,68 +0,0 @@
-<!DOCTYPE html>
```

Comment:
There is also a webapi/tct-filesystemapi-w3c-tests/filesystemapi/Metadata_modificationTime.html